bibedit/record_book.xml

```diff
@@ -20,7 +20,12 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 <!-- BibEdit-Template-Description: book -->
 <record>
   <datafield tag="020" ind1=" " ind2=" ">
-    <subfield code="a">VOLATILE:ISBN</subfield>
+    <subfield code="a">VOLATILE:ISBN (without dashes!)</subfield>
```

Comment:
I dont think so. If (for old books) we only have the 10digit version, one would have to calculate a new check digit. 
Florian 

-- 
Florian Schwennsen 
Deutsches Elektronen-Synchrotron DESY 
Building 01 Room O1.446 
phone: +49-40-8998-6190 

> From: "Samuele Kaplun" <notifications@github.com>
> To: "inspirehep/inspire" <inspire@noreply.github.com>
> Cc: "Florian Schwennsen" <florian.schwennsen@desy.de>
> Sent: Monday, 8 June, 2015 13:38:47
> Subject: Re: [inspire] bibedit: update record templates (#88)

> In bibedit/record_book.xml :
>> @@ -20,7 +20,12 @@ along with INSPIRE; if not, write to the Free Software
> > Foundation, Inc.,
> >  <!-- BibEdit-Template-Description: book -->
> >  <record>
> >    <datafield tag="020" ind1=" " ind2=" ">
> > -    <subfield code="a">VOLATILE:ISBN</subfield>
> > +    <subfield code="a">VOLATILE:ISBN (without dashes!)</subfield>

> shall we always recommend to put the 13digits one (978...)? Anyway we should
> write a BibCheck script that normalizes all isbn...

> —
> Reply to this email directly or view it on GitHub .